offline/offline.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Offline page</title>
+  <meta name="description" content="Offline page of this awesome website">
+  <meta name="author" content="Sarah Meysman">
+
+  <!--<link href="css/bootstrap.min.css" rel="stylesheet" />-->
+  <link href="/src/Frontend/Themes/Fork/Core/Layout/Css/screen.css?m=1593680880" rel="stylesheet" />
+
+</head>
+
+<body>
+<div class="container">
+  <div class="row justify-content-center mt-5">
+    <div class="col-md-8 col-xl-6">
+
+      <h1>You're offline</h1>
```

Comment:
wouldnt it be possible to use the translation system so these text are in the language you are visiting the site?